src/PowerShellEditorServices/Session/RemoteFileManager.cs

```diff
@@ -74,16 +74,22 @@ public class RemoteFileManager
 
             Register-EngineEvent -SourceIdentifier PSESRemoteSessionOpenFile {0}
 
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $false)
+            if ((Test-Path -Path 'function:\global:Open-EditorFile') -eq $false)
             {{
-                Set-Item -Path 'function:\global:PSEdit' -Value $PSEditFunction
+                Set-Item -Path 'function:\global:Open-EditorFile' -Value $PSEditFunction
+                Set-Alias psedit Open-EditorFile -Scope Global
             }}
         ";
 
         private const string RemovePSEditFunctionScript = @"
-            if ((Test-Path -Path 'function:\global:PSEdit') -eq $true)
+            if ((Test-Path -Path 'function:\global:Open-EditorFile') -eq $true)
             {
-                Remove-Item -Path 'function:\global:PSEdit' -Force
+                Remove-Item -Path 'function:\global:Open-EditorFile' -Force
+            }
+
+            if (Get-Alias psedit)
```

Comment:
It is unlikely that `psedit` wouldn't be defined but if it isn't this will generate a non-term error.  Maybe use `Test-Path alias:\psedit`?

Reply:
Fixed!